### ISSUE TITLE
Chestshop signs indestructible while TURN_OFF_SIGN_PROTECTION: true

### ISSUE THREAD

**Issue by NullCase** [ChestShop-authors/ChestShop-3#333]
#### Plugin Version
ChestShop's version is: 3.11 (build 215)
 
#### Plugin Config

Full config: https://gist.github.com/NullCase/dc015f806b9e3cdcc86ccda5ad716f84 
```
USE_BUILT_IN_PROTECTION: false
#Do you want to use built-in protection against chest destruction?
STICK_SIGNS_TO_CHESTS: false
#Do you want to have shop signs "stick" to chests?
TURN_OFF_DEFAULT_PROTECTION_WHEN_PROTECTED_EXTERNALLY: true
#EXPERIMENTAL: Do you want to turn off the default protection when another plugin is protecting the block? (Will leave the chest visually open - CraftBukkit bug!)
TURN_OFF_SIGN_PROTECTION: true
#Do you want to turn off the default sign protection? Warning! Other players will be able to destroy other people's shops!
TURN_OFF_HOPPER_PROTECTION: true
#Do you want to disable the hopper protection, which prevents the hoppers from taking items out of chests?
PROTECT_CHEST_WITH_LWC: false
#Do you want to protect shop chests with LWC?
PROTECT_SIGN_WITH_LWC: false
#Do you want to protect shop signs with LWC?
REMOVE_LWC_PROTECTION_AUTOMATICALLY: true
#Should the chest's LWC protection be removed once the shop sign is destroyed? 
```

edit: tried true/false for most of these options, to test. Also just broke signs on chests as a test. Also broke signs where I was the shop owner. As shop owner as a chestshop sign, the sign can be broken. As OP all chestshop signs can be broken.

#### Server Version
git-Paper-21 (MC: 1.16.1)
 
#### Server Log
nothing

#### What other plugins are you running?
![plugins](https://user-images.githubusercontent.com/32785664/86458261-fc58b180-bcf2-11ea-8c79-0d77fe5e1e96.png)

#### What is happening?
While sign protections are turned off, chestshop signs remain unbreakable.

#### What did you expect to happen?
Unprotected shop signs breakable, so the shop can be removed when it is abandoned.

### Additional context
I use GriefPrevention as the primary way to protect property for players. Chestshop allows us to transfer good securely. As shops are abandoned, players remove them. Since updating to 1.16 it seems like the shops have become indistructable.

**Comment by Phoenix616** [comment]
Fixed by efe9ff7, apparently I included the wrong issue number :S

**Comment by NullCase** [comment]
Tested this out today and it looks like you solved it. Thanks @Phoenix616
